src-ghc/Pact/Types/Crypto.hs

```diff
@@ -116,21 +115,19 @@ class (ConvertBS (PublicKey a),  Eq (PublicKey a),
 
 
 
-
 --------- HASHING ---------
 
-hashTx :: (H.HashAlgorithm a) => ByteString -> a -> Hash
-hashTx b algo = (Hash . B.convert . H.hashWith algo) b
 
-verifyHashTx :: (H.HashAlgorithm a) => Hash -> ByteString -> a -> Either String Hash
-verifyHashTx h b algo = if hashTx b algo == h
-  then Right h
+verifyHashTx :: PactHash -> ByteString -> Either String Hash
+verifyHashTx h b = if hashed == h
+  then Right (toUntypedHash h)
   else Left $ "Hash Mismatch, received " ++ show h
-       ++ " but our hashing resulted in " ++ show (hashTx b algo)
+       ++ " but our hashing resulted in " ++ show hashed
+  where hashed = hash b
 {-# INLINE verifyHashTx #-}
 
-initialHashTx :: (H.HashAlgorithm a) => a -> Hash
-initialHashTx algo = hashTx mempty algo
+initialHashTx :: Hash
```

Comment:
Same as previous comment. If keeping `hashTx` and `verifyHashTx`, then this is fine. But if delete `verifyHashTx`, this one should be deleted too.

Reply:
Removed